src/main/java/com/onlineshoppingsystem/project/controller/UserController.java

```diff
@@ -0,0 +1,49 @@
+package com.onlineshoppingsystem.project.controller;
+
+import com.onlineshoppingsystem.project.data.User;
+import com.onlineshoppingsystem.project.dto.UserDTO;
+import com.onlineshoppingsystem.project.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class UserController {
+
+    @Autowired
+    UserRepository userRepository;
+    private Long id;
+
+
+    @GetMapping(value = "users")
+    public ResponseEntity getAllUser(){return ResponseEntity.ok(userRepository.findAll());}
+
+    @GetMapping(value = "user/{id}")
+    public ResponseEntity getUserById(@PathVariable long id){
+        return userRepository.findAllById(id).orElseThrow(() -> new RuntimeException("User doesn't exist"));
```

Comment:
First, you need to change method from "findAllById"→(this method is from Iterable class) to "findById"  from Optional class. 
Second, if you want to return throw exception in this case, you need to change type of method from ResponseEntity to User.